dashboard/app/helpers/aichat_openai_helper.rb

```diff
@@ -72,6 +76,60 @@ def self.get_instructions(system_prompt, level_system_prompt, retrieval_contexts
     instructions
   end
 
+  # Reports and logs usage metrics to Cloudwatch
+  def self.report_usage_metrics(usage, messages, level_id, encrypted_channel_id, user_id)
+    return unless usage
+
+    filtered = messages.reject {|message| message[:content].is_a?(String)}
```

Comment:
For my understanding, what does this filter?

Reply:
Oh I think just filters the system prompt as this point; which makes me think it might be better to just convert the system prompt to use the same format as the other messages.